developer/src/kmc-keyboard-info/src/ttfmeta/README.md

```diff
@@ -0,0 +1 @@
+This was originally npmjs package ttfmeta. The source has been deleted from GitHub, so we will now using it locally so we can maintain it.
```

Comment:
> Remove ttfmeta from package.json, and copy only the relevant files into ttfmeta/ under kmc-keyboard-info. This is a stopgap measure as we will probably attempt to identify an alternate library in the future.

Instead of "maintain it", are we eventually aiming to replace it? Maybe also note the issue / related PR here?

Reply:
future issue #11641 (no need to add issue # in code at present I think)